Volunteer Information Pack (#63)
srobo/volunteer-emails: SR2023/2023-03-16-volunteer-information-pack.md

---
to: All volunteers for SR2023
subject: Your volunteer information for Student Robotics 2023
from: andys sr email
---

Hi all,

Firstly we're so thankful that you signed up to volunteer at Student Robotics 2023! I'm Andy, I'll be in charge of managing volunteering for the competition. If you have any questions, or have updates about your availability, please send me an email ASAP!

The competition is just round the corner, so I thought I'd share more detailed information about the event.

# Slack

> Before you read any further, we'd recommend you [make sure you are signed up to our slack chat room](https://goo.gl/forms/Maq41MHF8CYSRVn83). We usually use slack to communicate with volunteers during the competition itself.

Reviewer: ```suggestion
Before you read any further, we'd recommend you [make sure you are signed up to our Slack chat room](https://goo.gl/forms/Maq41MHF8CYSRVn83). We usually use Slack to communicate with volunteers during the competition itself.
```

# Safeguarding Policy

> **All volunteers are required to read the [safeguarding policy](https://opsmanual.studentrobotics.org/about-the-charity/safeguarding)**. Please ensure you have read this ahead of time, and e-mail safeguarding@studentrobotics.org to confirm you have read and understood it.

Reviewer: ```suggestion
**All volunteers are required to read the [safeguarding policy](https://opsmanual.studentrobotics.org/about-the-charity/safeguarding)**. Please ensure you have read this ahead of time, and email safeguarding@studentrobotics.org to confirm you have read and understood it.
```

# Set-up days

> We'll be having set-up days on the Thursday and Friday before the event, there is typically no set schedule for these days, just a list of tasks to get through. These lists are big though, we need all the help we can get to ensure everything is ready in time for the competition.

Reviewer 2: Suggestion: Add a CTA

```suggestion
We'll be having set-up days on the Thursday and Friday before the event, there is typically no set schedule for these days, just a list of tasks to get through. These lists are big though, we need all the help we can get to ensure everything is ready in time for the competition. If you're available, it would be great to get your help!
```

Reviewer 2: Suggestion: It might also be nice to note start times for those days, and potentially who to go to for said "tasks".

# Competition days
Please aim to arrive by 8:30am for both Saturday and Sunday if you can. This is when we will brief everyone on the day. If you arrive late, please try to find me (Andy Barrett-Sprot) to catch you up, and explain your roles to you.

> We're working out schedules for volunteers during the competition. We will assume you're available during all the timeslots you signed up for, so please make sure your responses are up-to-date! We'll soon be sending you an email with your individual roles and how to perform them, keep an eye out for it!

Reviewer 3: Maybe add something here about being able to edit your responses?

# Getting to the venue
Please check our external event page for how to get to the venue, and where to park: https://studentrobotics.org/events/sr2023/competition/

Unfortunately, there will a few factors to consider when trying to get to the venue:

### Rail Strikes

> There is a rail strike scheduled for the Thursday set-up day, and the Saturday of the competition. For both of these days, [SouthWestern Railway will have reduced services between Southampton and London](https://www.southwesternrailway.com/plan-my-journey/industrial-action), there are many rail operators in the UK though. At time of writing, not all have published plans for the competition weekend, so please check your intended train services are still running closer to the date.

Reviewer: ```suggestion
There is a rail strike scheduled for the Thursday set-up day, and the Saturday of the competition. For both of these days, [SouthWestern Railway will have reduced services between Southampton and London](https://www.southwesternrailway.com/plan-my-journey/industrial-action), there are many rail operators in the UK though. At time of writing, not all have published plans for the competition weekend, so please check your intended train services are still running closer to the date.
```
```suggestion
There is a rail strike scheduled for the Thursday set-up day, and the Saturday of the competition. For both of these days, [South Western Railway will have reduced services between Southampton and London](https://www.southwesternrailway.com/plan-my-journey/industrial-action), there are many rail operators in the UK though. At time of writing, not all have published plans for the competition weekend, so please check your intended train services are still running closer to the date.
```

### Southampton Marathon on Sunday
The [Southampton Marathon](https://www.southamptonmarathon.co.uk/residents-info) is scheduled on the competition Sunday. As a result, there will be a number of road closures in the area, including on the university campus where the competition is held. Parking will also be limited. Keep an eye on the [competition event page on our website](https://studentrobotics.org/events/sr2023/competition/) see up-to-date information on how to reach the competition. 

Last year the U1C bus still operated between the 'Southampton Airport Parkway' train station and the university campus, but the operator has not yet published their Marathon plans, so we'll keep you posted.

# Reimbursements for Accommodation and Travel 

> Sadly we aren't able to provide Accommodation for volunteers this year. However, we are able to reimburse up to £85 total or travel and accommodation. If you urgently need a reimbursement, you can claim reimbursement in our [reimbursement claim form](https://forms.gle/ATV3UeS9Sbd2ynhv8), and mark 'this reimbursement is urgent'.

Reviewer: ```suggestion
Sadly we aren't able to provide accommodation for volunteers this year. However, we are able to reimburse up to £85 total of travel and accommodation. If you urgently need a reimbursement, you can claim reimbursement in our [reimbursement claim form](https://forms.gle/ATV3UeS9Sbd2ynhv8), and mark 'this reimbursement is urgent'.
```

> ## COVID Safety

Reviewer 4: Is it intentional that the COVID Safety and Food section headings are level 2, implying that they're children of the Reimbursements section?

Author: oops good catch :)

> We take COVID safety very seriously, we'd like to strongly recommend volunteers wear masks, especially during set-up days, as we cannot afford a large number of volunteers quarantining themselves during the event

Reviewer 3: ```suggestion
We take COVID safety very seriously, we'd like to strongly recommend volunteers wear masks, especially during set-up days, as we cannot afford a large number of volunteers quarantining themselves during the event.
```

Author: looks like I fixed this while adding something else

> We take COVID safety seriously, we'd like to strongly recommend volunteers wear masks, especially during set-up days, as we cannot afford a large number of volunteers quarantining themselves during the event

Reviewer 5: People testing in advance is even more useful.

> We take COVID safety very seriously, We strongly recommend all volunteers test themselves before the event. Volunteers should also wear masks at all times, especially including set-up days as we cannot afford to lose a large number of volunteers during the event.

Reviewer 2: We should link to https://studentrobotics.org/covid-19/, and ensure we're being consistent

## Food during the weekend

> We're planning on arranging for sandwiches and fruit and biscuits to be available during lunchtimes. You indicated your dietary requirements during sign-ups so we'll do our best to accommodate for you. You should also have a long enough lunch break if you want to get something specific.

Reviewer 5: ```suggestion
We're planning on arranging for sandwiches and fruit and biscuits to be available during lunchtimes. If you indicated your dietary requirements during sign-ups so we'll do our best to accommodate for you. You should also have a long enough lunch break if you want to get something specific.
```

> We're planning on arranging for sandwiches and fruit and biscuits to be available during lunchtimes. If you indicated your dietary requirements during sign-ups so we'll do our best to accommodate for you. You should also have a long enough lunch break if you want to get something specific.

Reviewer 2: Issue: This sentence doesn't make much sense

```suggestion
We're planning on arranging for sandwiches and fruit and biscuits to be available during lunchtimes. If you indicated specific dietary requirements during sign-ups, we'll do our best to accommodate. You should also have a long enough lunch break if you want to get something specific.
```
